CMakeLists.txt

```diff
@@ -190,7 +190,7 @@ CHECK_C_SOURCE_COMPILES(
 
 CHECK_C_SOURCE_COMPILES(
   "#include <stddef.h>
-   int main(void) { int a,b; size_t m; __builtin_mul_overflow(a,b,&m); return 0; }"
+   int main(void) { int a=0, b=0; size_t m; __builtin_mul_overflow(a,b,&m); return 0; }"
```

Comment:
Harmless. Stop a spammy warning about using uninitialised values.

Reply:
Not harmless; they were intentionally uninitialized so they wouldn't be eliminated by an over eager optimizing compiler.

Reply:
? But this file is never executed. We are simply testing whether it compiles to an object file.

Reply:
AFAIK dead code elimination happens at compile time. If I remember correctly at keast with AIX's xlc it will result in a false positive for having `__builtin_mul_overflow` which then will result in a FTBFS problem.